core/src/main/scala/org/apache/spark/deploy/mesos/MesosDriverDescription.scala

```diff
@@ -40,24 +41,28 @@ private[spark] class MesosDriverDescription(
     val cores: Double,
     val supervise: Boolean,
     val command: Command,
-    val schedulerProperties: Map[String, String],
+    _schedulerProperties: Map[String, String],
```

Comment:
Update tag parameter.
@param schedulerProperties Extra properties to pass the Mesos scheduler